src/Poster/Poster.js

```diff
@@ -0,0 +1,115 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import { layoutOptions } from "./Poster.levers";
+import { Headline } from "./Poster.Headline";
+import { Content } from "./Poster.Content";
+import { Image } from "./Poster.Image";
+
+/**
+ * Poster Component
+ *
+ * HTML card element
+ */
+
+const PosterRoot = ({
+  bgImage: { src } = {},
+  bgGradient,
+  bgColor,
+  className,
+  children,
+  layout,
+}) => {
+  let wrapperClasses;
+  let imageWrapper;
+  let contentWrapper;
+
+  if (layout === "left") {
+    wrapperClasses = "su-flex su-flex-col su-justify-center md:su-flex-row";
+    imageWrapper =
+      "first:su-min-w-[14rem] su-rs-mb-2 su-mx-auto md:su-rs-mr-4 md:su-mb-0 md:su-ml-0";
+    contentWrapper = "last:su-items-start md:su-flex-grow";
+  } else {
+    wrapperClasses = "su-flex su-flex-col su-items-center";
+    contentWrapper = "last:su-items-center su-text-center";
+    imageWrapper = "first:su-rs-mb-2";
+  }
+
+  let gradientOverlay = "linear-gradient(to bottom, transparent, #181D1C)";
```

Comment:
I had set a default `linear-gradient` for the `gradientOverlay` variable in the event that the developer forgets to pass in a gradient.

Does it make sense to have a gradient option (lever)? 
Alternatively, we could remove the default gradient and have it as an empty string instead. That way it would be completely in the developer's hands to decide whether they want an overlay on their `bgImage`. Thoughts? 😄

Reply:
I was more thinking of an on/off lever for bg gradients vs options. What if the developer wants no gradient? Can you make it easy to remove?